script/Arcueid/Day 9/09_00_ARC09_2_2.txt

```diff
@@ -1,7 +1,7 @@
 [sha:5056bf205acc99d492cb9112d5be02c6ebe34244]{
 -- Page 10007, Offset 21492.
 -- やっぱり琥珀さんでいい。
-On second thought, I'd rather Kohaku-san do it.
+On second thought, I'd rather have Kohaku-san do it.
```

Comment:
Mild pref for original

Reply:
I prefer the new version. old version sounds a bit englandgo